src/meta-srv/src/bootstrap.rs

```diff
@@ -185,14 +187,14 @@ pub async fn metasrv_builder(
     plugins: Plugins,
     kv_backend: Option<KvBackendRef>,
 ) -> Result<MetasrvBuilder> {
-    let (kv_backend, election, lock) = match (kv_backend, opts.use_memory_store) {
+    let (kv_backend, election, lock) = match (kv_backend, &opts.backend) {
```

Comment:
This will deprecate the `use_memory_store` setting, should we add logic to make the metasrv store backward compatible in terms of this config

Reply:
It's ok. I believe `use_memory_store` should only be used for testing scenarios, so I don't think forward compatibility is necessary.